flink-table-store-dist/pom.xml

```diff
@@ -118,8 +112,6 @@ under the License.
                                     <include>org.apache.flink:flink-table-store-core</include>
                                     <include>org.apache.flink:flink-table-store-format</include>
                                     <include>org.apache.flink:flink-table-store-kafka</include>
-                                    <include>org.apache.flink:flink-connector-base</include>
```

Comment:
This is the reason? Can you post the exceptions?

Reply:
exception as follow:
```
Could not find any factories that implement 'org.apache.flink.table.store.shaded.org.apache.flink.connector.file.table.factories.BulkReaderFormatFactory' in the classpath.
    at org.apache.flink.table.factories.FactoryUtil.discoverFactory(FactoryUtil.java:526) 
```
the detailed reason is that flink-table-store use `FileFormatImpl` to discover formats implementing `BulkReaderFormatFactory` provided by Flink, but the name of interface has been shaded into different one, caused the exception as above